doc/tutorials.rst

```diff
@@ -47,10 +47,12 @@ algorithms using PennyLane and near-term quantum hardware.
 
 .. customgalleryitem::
     :tooltip: Do arbitrary state preparation on a real quantum computer.
+    :figure: ../examples/figures/NOON.png
     :description: :ref:`state_preparation`
 
 .. customgalleryitem::
     :tooltip: Use PennyLane to create a simple QGAN
+    :figure: ../examples/figures/gan.png
```

Comment:
I don't believe we have rights to use this image (looks like it was taken from internet)

Reply:
Check [here](https://arxiv.org/abs/1804.08641) for any images we could use

Reply:
Good catch.

Reply:
I made it myself but the one in the paper looks better :)

Reply:
@co9olguy do you have the original figure from this paper? we can use Figure 1b.